deb-package/build

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# Builds a Debian package for MXNET
+#
+# Copyright sociomantic labs GmbH 2017.
+# Distributed under the Apache License, Version 2.0.
+# (See accompanying LICENSE file or copy at
+# https://www.apache.org/licenses/LICENSE-2.0.txt)
+set -xeu
+
+# Set up package
+pkgname=libmxnet
+pkgversion_number=$(git describe --tags --dirty |
+                    sed 's/^v\(.*\)$/\1/' |
+                    sed 's/-\([0-9]\+\)-\(g[0-9a-f]\+\)/+\1~\2/' |
+                    sed 's/\(~g[0-9a-f]\+\)-dirty$/-dirty\1/' |
+                    sed 's/-dirty/~dirty.'`date +%Y%m%d%H%M%S`'/'
+                   )
+pkgdistro=$(lsb_release -cs)
+pkgversion="${pkgversion_number}~${pkgdistro}"
```

Comment:
> I have just one concern about using makd: it would be nice if our deb-creating solution was at least in principle upstreamable.

I don't agree really, packaging should be done by packagers, not upstream. To have a real debian compatible package there are tons of policies we are not going to comply.

I never knew upstreaming this was in the plans, but I wouldn't go that route.